src/librbd/exclusive_lock/PostAcquireRequest.cc

```diff
@@ -200,40 +199,13 @@ void PostAcquireRequest<I>::handle_open_image_cache(int r) {
   if (r < 0) {
     lderr(cct) << "failed to open image cache: " << cpp_strerror(r)
                << dendl;
-    send_close_image_cache();
```

Comment:
Wouldn't you still need to shut down the RWL cache dispatch layer?

Reply:
No need to shut down image cache here as it is done in InitRequest.